fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -195,12 +219,13 @@ public FHIRPersistenceJDBCImpl() throws Exception {
         ResourceTypesCache.setEnabled(fhirConfig.getBooleanProperty(PROPERTY_JDBC_ENABLE_RESOURCE_TYPES_CACHE,
                                       Boolean.TRUE));
 
```

Comment:
should we introduce a similar property for enabling/disabling the new ResourceReference caching?
or maybe we should remove all of these...i'm not sure why someone would turn them off (unless they found a bug in our stuff).

Reply:
+1 on the comment/question

Reply:
Created issue #1637. We cannot disable the cache in the new code - it is central to how some of the processing is performed. However, we should definitely replace the old cache stuff with the new cache.